lib/generators/shopify_app/app_proxy_controller/templates/app_proxy_controller.rb

```diff
@@ -0,0 +1,12 @@
+class AppProxyController < ApplicationController
+  if Rails.env.development? || Rails.env.test?
+   around_action :shopify_session
+  else
+   include ShopifyApp::AppProxyVerification
+  end
```

Comment:
Because

- I simply never knew that you could use a dev shop locally. 
- I thought dev shop was like any other shop except it didnt have a validity nor billing fn. attached to it.

Please point me to the documentation to use a dev shop locally.
Thanks in advance.